File: backend/main/src/main/java/me/onair/main/domain/story/repository/StoryRepository.java
package me.onair.main.domain.story.repository;

import me.onair.main.domain.story.entity.Story;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface StoryRepository extends JpaRepository<Story, Long> {
}

File: android/onair/domain/src/main/java/com/fm404/onair/domain/repository/story/StoryRepository.kt
package com.fm404.onair.domain.repository.story

import com.fm404.onair.domain.model.story.CreateStoryRequest
import com.fm404.onair.domain.model.story.Story

interface StoryRepository {
    suspend fun createStory(channelUuid: String, request: CreateStoryRequest): Result<Story>
}